File: Bronze/2015/December/FencePainting.py
a = 0
b = 0
c = 0
d = 0

with open("paint.in") as f:
    ab = f.readline()
    a = int(ab[0:ab.index(" ")])
    b = int(ab[ab.index(" ") + 1])

    count = 0
    for i in range(100):
        if (i >= a and i < b) or (i >= c and i < d):
            count += 1

    fp = open("paint.out", "w")
    fp.write(str(count))

File: Bronze/2015/December/FencePainting/FencePainting.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;

class FencePainting {
  private static int a;
  private static int b;
  private static int c;
  private static int d;

  private static void getInput() throws IOException {
    BufferedReader br = new BufferedReader(new FileReader("paint.in"));

    String ab = br.readLine();
    a = Integer.parseInt(ab.substring(0, ab.indexOf(" ")));
    b = Integer.parseInt(ab.substring(ab.indexOf(" ") + 1));

    String cd = br.readLine();
    c = Integer.parseInt(cd.substring(0, cd.indexOf(" ")));
    d = Integer.parseInt(cd.substring(cd.indexOf(" ") + 1));

    br.close();
  }

  private static int getUnitsPainted() {
    int count = 0;
    for (int i = 0; i < 100; i++) {
      if ((i >= a && i < b) || (i >= c && i < d)) {
        count++;
      }
    }
    return count;
  }

  public static void main(String[] args) throws IOException {
    FencePainting.getInput();
    FileWriter fw = new FileWriter(new File("paint.out"));
    fw.write(FencePainting.getUnitsPainted() + "\n");
    fw.close();
  }
}
